### PR TITLE
more tests for ContinueWhilePossible

### PR DESCRIPTION
**Pull request by ruchim**

### CODE REVIEW

File: src/main/resources/standardTestCases/cont_while_possible/cont_while_possible_scatter.wdl [hunk 1]
```diff
@@ -0,0 +1,24 @@
+task even_numbers_only {
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/cont_while_possible/cont_while_possible_scatter.wdl]: it's a little confusing that this *fails* for even numbers 🙂

File: src/main/resources/standardTestCases/cont_while_possible/cont_while_possible_scatter.wdl [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+	scatter(i in myRange){
+	  call even_numbers_only {input: i = i}
+	}
+
+	if (true) {
+	  call even_numbers_only as shouldSucceed { input: i = 1 }
+	  call even_numbers_only as shouldFail { input: i = 2 }
+	}
+}
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/cont_while_possible/cont_while_possible_scatter.wdl]: \n

File: src/main/resources/standardTestCases/cont_while_possible/cont_while_possible_scatter.wdl [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+workflow cont_while_possible_scatter {
+  Array[Int] myRange = range(4)
+
+	scatter(i in myRange){
+	  call even_numbers_only {input: i = i}
+	}
+
+	if (true) {
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/cont_while_possible/cont_while_possible_scatter.wdl]: why is this wrapped in an `if (true)`?

**Review by mcovarr — APPROVED** [review]

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/214/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/cont_while_possible/cont_while_possible_scatter.wdl (hunk 3 above)
Comment (ruchim) [reply src/main/resources/standardTestCases/cont_while_possible/cont_while_possible_scatter.wdl]: @mcovarr  Originally I expected this code to touch logic involving conditionals as well--but it should totally be unaffected.

**Comment by cjllanwarne** [comment]
Looks good to me! 👍 

Were you planning to add a conditionals tests too?

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/214/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by ruchim** [comment]
@cjllanwarne since the Cromwell changes were specific to shards--I added but then retracted a conditionals test case -- can you give an example of what kind of a scenario you'd like to see tested?

**Pull request merged**